master/buildbot/config.py

```diff
@@ -816,6 +816,34 @@ def __init__(self, name=None, slavename=None, startSlavenames=None, slavenames=N
 
         self.description = description
 
+        self.customBuildUrls = customBuildUrls or {}
```

Comment:
@campos-ddc we need the line to initialize, if the configuration passes None we wont get a failure I can remove it if we prefer a config failure